app/src/main/java/com/rootstrap/android/di/DI.kt

```diff
@@ -74,11 +69,8 @@ val dataModule = module {
     single<DispatcherProvider> { AppDispatcherProvider() }
     factory { Prefs(get()) }
     factory { UserRepository(get()) }
+    factory { ProductRepository() }
```

Comment:
Why repositories are `factory` and use cases are `single`? I mean why use cases are single

Reply:
Sure, I'll continue with single it has sense